src/styles/cards.scss

```diff
@@ -0,0 +1,130 @@
+.main {
+  display: flex;
+  justify-content: center;
+}
+
+.main__container {
+  display: flex;
+  align-self: center;
+  flex-wrap: wrap;
+  width: (4 * $card-width) + (3 * $card-margin) + (2 * $inside-padding);
+  padding: 50px 0;
+}
```

Comment:
move this block to a separate file

Reply:
comment wasn't fixed 
![image](https://user-images.githubusercontent.com/122989585/224481152-53b6052e-c46f-494e-8ac5-d102c3d6aad3.png)